app/models/techhire_location.rb

```diff
@@ -5,7 +5,7 @@ class TechhireLocation < ActiveRecord::Base
 
   geocoded_by :full_address
 
-  after_validation :geocode, if: ->(obj){(obj.longitude.blank? || obj.latitude.blank?)} || ->(obj){(obj.city_changed? || obj.state_changed?)}
+  after_validation :geocode, if: ->(obj){ (!Rails.env.test? || obj.longitude.blank? || obj.latitude.blank?)} || ->(obj){(obj.city_changed? || obj.state_changed?)}
```

Comment:
@scottalexandra i think we determined that this wasn't the culprit after all, but rather the FactoryGirl.lint thing. if that's the case it might be better to take this out so it doesn't cause confusion later